.gitignore

```diff
@@ -20,3 +20,6 @@
 /public/packs
 /public/packs-test
 /node_modules
+
+.ruby-gemset
+.ruby-version
```

Comment:
Do we want this project to depend on a specific Ruby version? If so, then we should check this in, right? If not, then I see nothing wrong with ignoring it. Check out this SO https://stackoverflow.com/questions/16087120/should-i-check-in-ruby-gemset-and-or-ruby-version

I know Heroku wants us to specify the Ruby version. Does this mean we need to keep .ruby-version?

Reply:
We do want to depend on a specific version of Ruby. Heroku will want us to specify that in the Gemfile though. The only reason that I had git ignore `.ruby-version` is because another open source Rails project I'm working on did that.

But when I think about it, I don't think that there is any harm in checking it in. If people use RVM or rbenv it will be there for them. If they don't use it, it just won't have any effect. I'll go a head and check it in.